decentraland/editor-electron
# Add missing scenes feature

* Add snackbar notifications
* Add Missing Scenes feature to allow reimporting/discarding scenes

How to test:
* Add a broken path to the `config.json` file in the app home directory (either manually or by importing a project and deleting the directory afterward)
* Reload app
* A notification should be shown in the snackbar section with two options: view & discard all
* Discarding all should remove the notification and also remove all the broken paths from `config.json`
* If you click on "view", a modal should open showing the broken paths and an option to fix them/discarding them one by one

Snackbar:
<img width="571" alt="Screenshot 2024-07-22 at 1 42 31 PM" src="https://github.com/user-attachments/assets/d9307cc8-bb93-4d36-ac31-81a130731750">

Missing Scenes modal:
<img width="592" alt="Screenshot 2024-07-22 at 1 44 32 PM" src="https://github.com/user-attachments/assets/65bb11c3-8252-454e-9e3d-5481220d73b4">
